coder_sniffer/Drupal/Test/Commenting/DeprecatedUnitTest.inc

```diff
@@ -2,9 +2,9 @@
 
 /**
  * @file
+ * This file contains test cases for DeprecatedSniff.
```

Comment:
These changes are to make the test file clean for all coding standards (but I think on reflection this was unnecessary as the test directories are ignored. Oh well ...)